example_verifyBlob_test.go

```diff
@@ -55,7 +55,7 @@ func Example_verifyBlob() {
 	// createTrustStoreForBlobVerify creates a trust store directory for demo purpose.
 	// Users could use the default trust store from Notary Project and add trusted
 	// certificates into it following the trust store spec:
-	// https://github.com/notaryproject/notaryproject/blob/v1.1.0/specs/trust-store-trust-policy.md#trust-store
+	// https://github.com/notaryproject/specifications/tree/9c81dc773508dedc5a81c02c8d805de04f65050b/specs/trust-store-trust-policy.md#trust-store
```

Comment:
If it has a tag, use tag instead of the digest.

Reply:
As we haven't released the blob spec yet, this will be a digest for now.